src/mem/Allocator.cc

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (C) 1996-2022 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#include "squid.h"
+#include "mem/Allocator.h"
+
+size_t
+Mem::Allocator::RoundedSize(size_t s)
+{
+    return ((s + sizeof(void*) - 1) / sizeof(void*)) * sizeof(void*);
+}
```

Comment:
Please consider inlining this trivial no-dependencies method, especially if doing so eliminates the need for Allocator.cc.